src/styles/star.css

```
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  height: 16px;
+  width: 16px;
+  justify-content: space-between;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card__stars-stars--1 :nth-child(-n + 1),
+.card__stars-stars--2 :nth-child(-n + 2),
+.card__stars-stars--3 :nth-child(-n + 3),
+.card__stars-stars--4 :nth-child(-n + 4),
+.card__stars-stars--5 :nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
```

```suggestion
}

```
And change file's name to stars